app/workers/education_form/process10203_submissions.rb

```diff
@@ -180,23 +163,23 @@ def format_application(data)
       nil
     end
 
-    def remaining_entitlement_days(gi_bill_status)
-      months = gi_bill_status.remaining_entitlement.months
-      days = gi_bill_status.remaining_entitlement.days
+    def remaining_entitlement_days(remaining_entitlement)
+      months = remaining_entitlement.months
+      days = remaining_entitlement.days
       months * 30 + days
     end
 
     # Inverse of less than six months check performed in EducationForm::SendSchoolCertifyingOfficialsEmail
-    def more_than_six_months?(gi_bill_status)
-      return true if gi_bill_status.remaining_entitlement.blank?
+    def more_than_six_months?(remaining_entitlement)
+      return false if remaining_entitlement.blank?
```

Comment:
I flipped this because having `gi_bill_status == {} || gi_bill_status.remaining_entitlement.blank?` as true was marking the form as PROCESSED